pwiz_tools/Skyline/Controls/Graphs/AreaReplicateGraphPane.cs

```diff
@@ -822,6 +835,26 @@ private string GetDotProductResultsText(TransitionGroupDocNode nodeGroup, int in
                         values = replicateIndices.Select(nodeGroup.GetIsotopeDotProduct);
                     }
                     break;
+                case AreaExpectedValue.ratio_heavy:
+                    var document = GraphSummary.DocumentUIContainer.DocumentUI;
+                    var normalizeOption = AreaGraphController.AreaNormalizeOption.Constrain(document.Settings);
+                    values = new float?[] { };
+                    if (normalizeOption.NormalizationMethod is NormalizationMethod.RatioToLabel ratioToHeavy)
+                    {
+                        var precursorNodePath = DocNodePath.GetNodePath(nodeGroup.Id, document);
+                        if (precursorNodePath.Peptide != null && nodeGroup.LabelType.Equals(IsotopeLabelType.light))

```

Comment:
I think this should be "!nodeGroup.LabelType.Equals(ratioToHeavy.Label)"
"ratioToHeavy.Label" is the label that is going to appear in the numerator.
If nodeGroup.LabelType is different from ratioToHeavy.Label, then you want to display the rdotp.